From f46459170a8924f8fc23174c21e51b44b29cda14 Mon Sep 17 00:00:00 2001
From: Phillip Lougher <phillip@squashfs.org.uk>
Date: Wed, 27 Aug 2014 06:14:38 +0100
Subject: [PATCH 079/120] mksquashfs: fix usage of "_" in new options

Gah, I like using "_" as the separator between words in
options.  But, a long time ago I noticed I'd ended up with "-"
as the separator in the majority case, and so I'm stuck with
"-" as the separator.

Fix verbose_option, true_option, false_option, action_file
and verbse_action_file.

Signed-off-by: Phillip Lougher <phillip@squashfs.org.uk>
---
 squashfs-tools/mksquashfs.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/squashfs-tools/mksquashfs.c b/squashfs-tools/mksquashfs.c
index 0dd4a48..607d70c 100644
--- a/squashfs-tools/mksquashfs.c
+++ b/squashfs-tools/mksquashfs.c
@@ -5197,7 +5197,7 @@ int main(int argc, char *argv[])
 			if(res == 0)
 				exit(1);
 
-		} else if(strcmp(argv[i], "-verbose_action") == 0 ||
+		} else if(strcmp(argv[i], "-verbose-action") == 0 ||
 				strcmp(argv[i], "-va") ==0) {
 			if(++i == argc) {
 				ERROR("%s: %s missing action\n",
@@ -5208,7 +5208,7 @@ int main(int argc, char *argv[])
 			if(res == 0)
 				exit(1);
 
-		} else if(strcmp(argv[i], "-true_action") == 0 ||
+		} else if(strcmp(argv[i], "-true-action") == 0 ||
 				strcmp(argv[i], "-ta") ==0) {
 			if(++i == argc) {
 				ERROR("%s: %s missing action\n",
@@ -5219,7 +5219,7 @@ int main(int argc, char *argv[])
 			if(res == 0)
 				exit(1);
 
-		} else if(strcmp(argv[i], "-false_action") == 0 ||
+		} else if(strcmp(argv[i], "-false-action") == 0 ||
 				strcmp(argv[i], "-fa") ==0) {
 			if(++i == argc) {
 				ERROR("%s: %s missing action\n",
@@ -5230,7 +5230,7 @@ int main(int argc, char *argv[])
 			if(res == 0)
 				exit(1);
 
-		} else if(strcmp(argv[i], "-action_file") == 0 ||
+		} else if(strcmp(argv[i], "-action-file") == 0 ||
 				strcmp(argv[i], "-af") ==0) {
 			if(++i == argc) {
 				ERROR("%s: %s missing filename\n", argv[0],
@@ -5240,7 +5240,7 @@ int main(int argc, char *argv[])
 			if(read_action_file(argv[i], ACTION_LOG_NONE) == FALSE)
 				exit(1);
 
-		} else if(strcmp(argv[i], "-verbose_action_file") == 0 ||
+		} else if(strcmp(argv[i], "-verbose-action-file") == 0 ||
 				strcmp(argv[i], "-vaf") ==0) {
 			if(++i == argc) {
 				ERROR("%s: %s missing filename\n", argv[0],
-- 
2.8.3

